env0/resource_aws_credentials.go

```diff
@@ -14,6 +15,8 @@ func resourceAwsCredentials() *schema.Resource {
 		ReadContext:   resourceAwsCredentialsRead,
 		DeleteContext: resourceAwsCredentialsDelete,
 
+		Importer: &schema.ResourceImporter{StateContext: resourceAwsCredentialsImport},
```

Comment:
@TomerHeber if we're already at it, can we perhaps also support import for all other credentials type? I'd do it in a separate PR tho

Reply:
keeping the issue open.